modules-js/react-fleet/src/form-elements/inputs/MemorableDateInput.tsx

```diff
@@ -0,0 +1,458 @@
+import React from 'react';
+
+import { css } from 'emotion';
+
+interface Props {
+  initialDate?: string;
```

Comment:
should this be a `string` or a `Date`? I think the latter for symmetry with this component’s output.

Reply:
I wanted to accept strings for `initialDate`, `earliestDate`, and `latestDate` because `<input type="date">` accepts strings for its correlating attributes `value`, `min`, and `max` and it seems appropriate to follow a similar pattern. Now that I’m talking about it “out loud” though, I wonder if it’s better to have the property names match those of the native input?

Reply:
The way I'm thinking of it is that this component can be used to give me `Date` objects (via the `handleDate` callback), which I’ll then store in various model objects.

When it comes time to edit one of my models, I’ll want to pass its `Date` back into the editor to display.

That internal consistency feels more important to me than matching an HTML spec.

Reply:
I can see that. I’ll change the component to accept `string | Date` and allow for either case.